handbook/customers.md

```diff
@@ -1,63 +1,63 @@
 
 L3: Test the Zoom Meeting link to ensure that it is working.
 
 L4: Ensure that agenda documents are attached and accessible to meeting attendees (as appropriate to the situation).
 
 #### Day of meeting
 
-L1: Join meeting 2-3 minutes prior to the meeting start time.
+L1: Join the meeting two to three minutes before the meeting start time.
 
-L2: If the customer does not join the call after 3-minutes, contact the customer with:
+L2: If the customer does not join the call after three minutes, contact the customer by:
 
 - Slack, if we have a shared channel.
-- Email, using the email address from the calendar invite.
+- email, using the email address from the calendar invite.
 - LinkedIn, send a direct message.
-- Phone, try finding their number to text and/or call (as appropriate to the device type: landline vs cell phone).
+- phone, try finding their number to text and/or call (as appropriate to the device type: landline vs. cell phone).
 
-L3: In these communications to the customer remember to approach the situation with empathy. Anything could have happened.
+L3: In these communications to the customer, remember to approach the situation with empathy. Anything could have happened.
 
-L4: Be ready to supply an alternative date and time to reschedule the call. Suggest 2-3 options for the customer to choose from.
+L4: Be ready to supply an alternative date and time to reschedule the call. Suggest two to three options from which the customer can choose.
 
 L5: Ensure that contact information is accurate and that meeting invites can be received and accessed by the customer.
 
-L6: Repeat back to the customer the newly agreed upon date and time, as well as the contact information.
+L6: Repeat back to the customer the newly agreed-upon date and time, and the contact information.
 
 L7: Congratulations, you’re ready to set up a new call.
 
-### Next steps after a customer conversation
-After a customer conversation, it can sometimes feel like there are 1001 things to do, but it can be hard to know where to start.  Here are some tips:
+### Next steps after a customer conversation 
+After a customer conversation, it can sometimes feel like there are 1,001 things to do, but it can be hard to know where to start.  Here are some tips: 
 
 
 ## Support process
 
 This section outlines the customer and community support process at Fleet.
 
-- L1: Basic help desk resolution and service delivery -> CS team handles these with occasional support from L2
-- L2: In-depth technical suppport -> CS team with L2 oncall technician
-- L3: Expert product and service support -> CS team liases with L2 and L3 oncall technician.
+- L1: Basic help desk resolution and service delivery -> CS team handles these with occasional support from L2 
+- L2: In-depth technical support -> CS team with L2 onc-all technician 
+- L3: Expert product and service support -> CS team liases with L2 and L3 on-call technician
```

Comment:
@tgauda, in reference to the use of "CS team," this is not explained on this page, but I do later see "Customer Success team." Are these the same? If so, for clarity and translation purposes, we want to write this out and avoid acronyms in general.